pkgs/tools/misc/uutils-coreutils/default.nix

```
@@ -7,29 +7,26 @@
 , Security
 , prefix ? "uutils-"
 , buildMulticallBinary ? true
+, unstableGitUpdater
 }:
 
+assert prefix != null;
+
 stdenv.mkDerivation rec {
   pname = "uutils-coreutils";
-  version = "0.0.6";
+  version = "unstable-2021-06-17";
 
   src = fetchFromGitHub {
     owner = "uutils";
     repo = "coreutils";
-    rev = version;
-    sha256 = "sha256-dnswE/DU2jCfxWW10Ctjw8woktwWZqyd3E9IuKkle1M=";
+    rev = "1f4d5827ec19f070d8be11603d47a40a504b3a87";
+    sha256 = "sha256-ys5IyFToQsI+EHm9mG4ObQBsFhTACHxjWV2MlDoXpUM=";
   };
 
-  postPatch = ''
-    # can be removed after https://github.com/uutils/coreutils/pull/1815 is included
-    substituteInPlace GNUmakefile \
-      --replace uutils coreutils
-  '';
-
   cargoDeps = rustPlatform.fetchCargoTarball {
     inherit src;
     name = "${pname}-${version}";
-    hash = "sha256-92BHPSVIPZLn399AcaJJjRq2WkxzDm8knKN3FIdAxAA=";
+    hash = "sha256-M3v2Gurj+zjV4Vx9jK51bADH9gh7ZyZ1FOncWkUOwCE=";
```

I get a hash mismatch.

```suggestion
    hash = "sha256-Mmj+7OaW+dhuC4rMg4LnmnHned4H8Ulp4gIbR0pHlV4=";
```

---

The unstable updater has some problem and always produces the wrong vendorSha.